locale/fi/toast.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ilmoitus (Toast)",
+  "intro": "Ilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää tietoa käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Toast-ilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ilmoituksesta.",
+    "Ilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan.",
+    "Sivulla voi olla yhtä aikaa useita Toast-ilmoituksia näkyvissä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Komponentti koostuu kuvaavasta ikonista, otsikosta ja pääsisällöstä. Komponentissa ei ole eri tiloja (status variant)."
```

Comment:
I don't think the lack of status variants needs to be stated.

Reply:
Fixed.

Reply:
Great. One more thought for the same line that occurred to me: is the word "kuvaavasta" really necessary there? Also, should we mention that the heading is optional?

Reply:
fixed.